content/community/gsoc/2021/ideas.html

```diff
@@ -376,25 +368,23 @@ <h4><img src="/images/App_Generic_32.png"/>
 </ul>
 
 
-<h4><img src="/images/App_Generic_32.png"/>Adding write support for more filesystems</h4>
+<h3><img src="/images/App_Generic_32.png"/>Adding write support for more filesystems</h3>
 
-<h5>Idea description</h5>
+<h4>Idea description</h4>
 
 <p>Some filesystems can only be read, but not written, from Haiku. The goal of this idea is to add write support for one of these filesystems (of your choice, from the list below).</p>
 
 <ul>
-<li><a href="http://en.wikipedia.org/wiki/Xfs">XFS</a> (<a href="http://xfs.org/index.php/Main_Page">Development community</a>, <a href="http://oss.sgi.com/projects/xfs/index.html">homepage</a>) is a filesystem originally developed for the IRIX operating system. Today it is commonly used by Haiku developers who build Haiku from Linux, because of its better support for extended filesystem attributes (unlike ext4).
-
-	Update: You are recommended to look at the current status of XFS. We are in the process of merging XFS v2. XFS v3 has been out for a long time and bringing read support for it could be a better idea before jumping to providing write support. This may require some discussion and we encourage you to reach out to us.</li>
+<li><a href="http://en.wikipedia.org/wiki/Xfs">XFS</a> (<a href="http://xfs.org/index.php/Main_Page">Development community</a>, <a href="http://oss.sgi.com/projects/xfs/index.html">homepage</a>) is a filesystem originally developed for the IRIX operating system. Today it is commonly used by Haiku developers who build Haiku from Linux, because of its better support for extended filesystem attributes (unlike ext4).</li>
```

Comment:
here you have removed a previous update.

also please add linebreaks when you change long lines like this, because it is impossible to read the changes in github